### ISSUE TITLE
Reflection

### ISSUE THREAD

**Issue by sbabicki** [UBC-MDS/532_Group_22#70]
In this section, your group should document on what you have implemented in your dashboard so far and explain what is not yet implemented. It is important that you include what you know is not working in your dashboard, so that your TAs can distinguish between features in development and bugs. Since this is the last milestone, you really need to motivate well why you have not chosen to include some feature that you were planning on including previously.

This week it is suitable to include thoughts on the feedback you received from your peer and/or TA, e.g.

- Has it been easy to use your app?
- Are there reoccurring themes in your feedback on what is good and what can be improved?
- Is there any feedback (or other insight) that you have found particularly valuable during your dashboard development?
This section should be around 300-500 words and the reflection-milestone4.md document should live in your GitHub.com repo in the doc folder.

**Comment by sbabicki** [comment]
Thanks @CalSchafer for saving me here! Let me know on slack when you are done with the first revision and I can go through and see if there is anything else to add.

**Comment by sbabicki** [comment]
There are a few things from closed issues that we made the desicion to not implement that we need to make sure we mention in the reflection: 

- [x] TA feedback #66 Nice touch adding a graph save option. It would be helpful if you made a note to let the user know they can do this. It was hard to notice. - _This is an altair feature and not something we are really advertising, so we are leaving it in but not drawing attention to it_
- [x] TA feedback #66 For tab 2: As I was using this tab, I found it a little frustrating that my location selection was lost when I toggled between the Province and CMA options. - Thank you for defining CMA in your reflection for me! Otherwise, well done on this tab, the selections work, and the graphs are well done. - _Decided not to implement saving selections as it would be too dificult to implement. Instead set a better default selection._
- [ ] Bug: We aren't really dealing with missing data in a smart way. Future versions of the app should seek to resolve this. 
- [x] #107 Design upgrades
